packages/frontend/src/components/VisualizationConfigs/ChartConfigPanel/Series/Series.styles.ts

```diff
@@ -1,39 +1,5 @@
-import { Colors, FormGroup, HTMLSelect } from '@blueprintjs/core';
-import styled, { css } from 'styled-components';
-
-const InputStyle = css`
-    background: ${Colors.WHITE};
-    border: 0.7px solid ${Colors.LIGHT_GRAY1};
-    box-sizing: border-box;
-    box-shadow: inset 0px 1px 1px rgba(16, 22, 26, 0.2);
-`;
-
-const FlexTemplate = css`
-    display: flex;
-    flex-direction: column;
-`;
-
-export const GroupSeriesBlock = styled.div`
-    ${FlexTemplate}
-`;
-
-export const GroupSeriesWrapper = styled.div`
-    padding: 5px;
-    background: ${Colors.LIGHT_GRAY5};
-    border-radius: 0.286em;
-`;
-
-export const GroupedSeriesConfigWrapper = styled.div`
-    margin-bottom: 0.357em;
-`;
-
-export const GroupSeriesInputs = styled.div`
-    display: flex;
-    flex: 1;
-    gap: 0.714em;
-    justify-content: space-between;
-    margin-bottom: 0.714em;
-`;
+import { Colors } from '@blueprintjs/core';
+import styled from 'styled-components';
 
 export const SeriesBlock = styled.div`
```

Comment:
this is the only one that's left. can we do it here or is it going to be done somewhere else?

Reply:
Just posted the final review with the final cleanup.

Reply:
sweet!